ruling/snapshots/prefer-object-literal

```diff
@@ -0,0 +1,27 @@
+src/angular.js/i18n/src/converter.js: 38
+src/angular.js/src/ngAnimate/animateCss.js: 600
+src/brackets/src/extensibility/ExtensionManager.js: 313
+src/brackets/src/extensions/default/HealthData/HealthDataManager.js: 53
+src/brackets/src/extensions/default/InlineColorEditor/ColorEditor.js: 557,570,582
+src/brackets/src/extensions/default/JavaScriptRefactoring/RefactoringUtils.js: 365
+src/brackets/src/LiveDevelopment/LiveDevServerManager.js: 97
+src/Ghost/core/server/helpers/index.js: 1
+src/Ghost/core/server/helpers/navigation.js: 64
+src/Ghost/core/server/lib/image/image-size.js: 23
+src/Ghost/core/server/update-check.js: 87
+src/jest/packages/pretty-format/perf/test.js: 182
+src/jquery/external/sinon/sinon.js: 4500
+src/react-native/Libraries/Components/Subscribable.js: 22
```

Comment:
I think we should exclude cases like this. How about instead of ignoring only functions, ignore statements spreading over more than 1 line?